src/software/field_tests/field_test_fixture.py

```diff
@@ -336,6 +332,20 @@ def load_command_line_arguments():
         help="Run the test with friendly robots in yellow mode",
     )
 
+    estop_group = parser.add_mutually_exclusive_group()
+    estop_group.add_argument(
+        "--keyboard_estop",
+        action="store_true",
+        default=False,
+        help="Allows the use of the spacebar as an estop instead of a physical one",
+    )
+    estop_group.add_argument(
+        "--disable_estop",
+        action="store_true",
+        default=False,
+        help="Disables checking for estop plugged in (ONLY USE FOR LOCAL TESTING)",
+    )
```

Comment:
I'm wondering if disabling the e-stop is needed anymore, now that the spacebar can be used as a default. I'm not sure when we would have scenarios where we want to disable the e-stop, but also not have access to a keyboard. Could you change it so that if there's no e-stop plugged in, it defaults to using the spacebar, and maybe a warning message, or in theunderscope, the e-stop status displays what  is controlling it.

Reply:
This is a good point. I think we should keep it for scenarios where we want to run thunderscope without worrying that commands are accidentally are going to be sent out to the robot. We should rename it to something like `--disable_communication`